CMSIS/NN/Include/arm_nnfunctions.h

```diff
@@ -1696,7 +1696,7 @@ extern    "C"
    * @param[in,out]   src                pointer to input tensor
    * @param[in]       bufferA            temporary buffer used for optimization and is necessary when
    *                                     ARM_MATH_DSP is defined.
-   *                                     Required space: (ch_src * dim_dst_width) * sizeof(q15_t) bytes
+   *                                     Required space: ch_src * dim_dst_width bytes
```

Comment:
Lets remove the 'Required space: ch_src * dim_dst_width bytes' completely from this file so it is one less file to keep track of. . And lets remove 'temporary buffer used for optimization and is necessary when ARM_MATH_DSP is defined.' as well. and replace it with something like 'Temporary buffer used for optimization. Use arm_avgpool_s8_get_buffer_size() to get the size of required memory'

Reply:
Good point